sdk/script/acquisition-sdk.ts

```diff
@@ -169,7 +172,17 @@ export class AcquisitionManager {
                     return;
             }
         }
-
+        
+        if (fromLabelOrAppVersion) {
```

Comment:
Do we need to make this check since setting the property to undefined/null would have the affect of not creating the property at all.